src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Websites</PackageId>
     <Description>Provides website management capabilities for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.Websites</AssemblyName>
-    <VersionPrefix>1.6.0-preview</VersionPrefix>
+    <VersionPrefix>1.7.0-preview</VersionPrefix>
```

Comment:
@Nking92 add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes